modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1735,10 +1735,10 @@ protected int getCellIndex(T cell){
      *                                                                         *
      **************************************************************************/
 
-    final VirtualScrollBar getHbar() {
+    protected final VirtualScrollBar getHbar() {
```

Comment:
`com.sun.javafx.scene.control.VirtualScrollBar` is private, so unless I'm missing something, adding `protected` here won't be of any use.

Either you move the control to public `javafx.scene.control.VirtualScrollBar`, or you return the superclass `protected final ScrollBar getHbar()`?

I guess the latter is good enough, as usually there is no need to modify the current implementation.